backend/src/utils/resourceUtils.ts

```diff
@@ -905,6 +910,64 @@ export const migrateTemplateDisablement = async (
     });
 };
 
+/**
+ * As we release Dashboard v2.24.0 we will be re-enabling Trusty for RHOAI, but not for Bias Metrics.
+ * This means we will need to make sure Bias is in a state of disabled while potentially the client
+ * reads Bias as
```

Comment:
Looks like there was more to this sentence?

Reply:
hahah, I'll fix.